content/docs/integrating-with-other-libraries.md

```diff
@@ -39,21 +39,22 @@ class SomePlugin extends React.Component {
 }
 ```
 
-Note that we defined both `componentDidMount` and `componentWillUnmount` [lifecycle methods](/docs/react-component.html#the-component-lifecycle). Many jQuery plugins attach event listeners to the DOM so it's important to detach them in `componentWillUnmount`. If the plugin does not provide a method for cleanup, you will probably have to provide your own, remembering to remove any event listeners the plugin registered to prevent memory leaks.
+Tenga en cuenta que definimos `componentDidMount` y` componentWillUnmount` [métodos del ciclo de vida](/docs/react-component.html#the-component-lifecycle). Muchos plugins de jQuery adjuntan listeners de eventos al DOM, por lo que es importante desmontarlos en `componentWillUnmount`. Si el complemento no proporciona un método para la limpieza, probablemente tendrás que proporcionar el tuyo, recordando eliminar cualquier listener de eventos que el plugin haya registrado para evitar pérdidas de memoria.
```

Comment:
We are refering to the user as `tú` instead of `usted` .Maybe you can change `Tenga` by `Ten`